phala-cloud/confidential-ai/overview.mdx

```diff
@@ -1,24 +1,43 @@
 ---
-description: Run Large Language Models securely in GPU Trusted Execution Environment
-  (TEE) for confidential AI.
+description: Run Large Language Models securely in GPU TEE for confidential AI.
 title: Overview
 ---
 
-
-## Overview
-
 Private AI or called confidential AI addresses critical concerns such as data privacy, secure execution, and computation verifiability, making it indispensable for sensitive applications. As illustrated in the diagram below, people currently cannot fully trust the responses returned by LLMs from services like OpenAI or Meta, due to the lack of cryptographic verification. By running the LLM inside a TEE, we can add verification primitives alongside the returned response, known as a Remote Attestation (RA) Report. This allows users to verify the AI generation results locally without relying on any third parties.
 
 <Frame>
     <img src="/images/compare-llm-with-tee-or-not.png" alt="Compare LLM with TEE or not" />
 </Frame>
 
-## Implementation
+Phala Cloud provides two major products to provide GPU TEE for confidential AI.
+
+<Columns cols={2}>
+  <Card
+    icon="bot"
+    href="/phala-cloud/confidential-ai/confidential-model/confidential-ai-api"
+    title="Confidential Models"
+  >
+    Run Large Language Models in GPU TEE
+  </Card>
+  <Card
+    icon="microchip"
+    title="Confidential GPU"
+    href="/phala-cloud/confidential-ai/confidential-gpu/overview"
+  >
+    Rent GPU TEE Server to run your own model.
+  </Card>
+</Columns>
+
+**Confidential Models** provide various models to provide hassle-free integration using its OpenAI-compatible format. Deploy seamlessly across cloud, on-premises, or edge environments.
+
+And if you want to have GPU TEE server to serve your owen GPU service under the protection of TEE, you should choose **Confidential GPU** service. 
+
+### Private ML SDK
```

Comment:
Let's use the title "Open Source" as the subtitle. We can say all the Confidential AI features are supported by dstack. Note that it's not private-ml-sdk, because dstack is the full framework and private-ml-sdk is just the plugin to dstack.